socrata/lazy_pool.py

```diff
@@ -41,7 +41,10 @@ def _make_worker(self, predicate):
         def _w():
             with self.thread_sem:
                 for thing in self.iterable:
-                    self.result_queue.put(predicate(thing))
+                    try:
+                        self.result_queue.put(predicate(thing))
+                    except Exception as e:
+                        self.result_queue.put(e)
```

Comment:
What happens when `predicate` (predicate?) returns an instance of Exception normally?

Reply:
yep, it'd behave weirdly. but i was fine with that when i wrote this :angel: